CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changes in Fbx Exporter
 
+## [UNRELEASED] - 2020-09-29
+### Changed
+- Use DeepConvertScene from Autodesk® FBX SDK to convert from Unity (left handed system with odd parity) to Maya (right handed system with odd parity) axis system.
+    - Previously the conversion was performed by the FBX exporter.
```

Comment:
For the sub-bullet can you explain what effect it has on my code if I'm updating?

Reply:
(Either very briefly or by referring to the documentation)